examples/rl_gsm8k/cot_math_agent.py

```diff
@@ -62,18 +67,53 @@ def parse_completion(self, completion: str, prompt_id: str) -> Generator[Step, N
             return
         yield step
 
+    def make_prompt(self, agent: Any, tape: Tape) -> Prompt:
+        """Create a prompt from tape interactions.
+
+        This method constructs a prompt by processing the tape content and agent steps description
+        into a format suitable for LLM consumption. It includes token count checks and tape trimming
+        if needed to fit within context size limits.
+
+        Args:
+            agent (Any): The agent object containing LLM configuration.
+            tape (Tape): The tape object containing interaction history.
+
+        Returns:
+            Prompt: A Prompt object containing formatted messages for LLM consumption.
+
+        Note:
+            The method performs the following steps:
+
+            1. Cleans the tape content
+            2. Gets steps description
+            3. Converts tape to messages
+            4. Checks token count and trims if needed
+            5. Reconstructs messages if trimming occurred
+        """
+        cleaned_tape = self.prepare_tape(tape)
+        steps_description = self.get_steps_description(tape, agent)
+        messages = self.tape_to_messages(cleaned_tape, steps_description)
+        messages = self.tape_to_messages(cleaned_tape, steps_description)
```

Comment:
We don't need to copy-paste all this code. Let's simplify.

Reply:
I imagine that all you need in this particular case is just to make the one and only user message from the one and only step 0 of the tape.